torchvision/prototype/transforms/_transform.py

```diff
@@ -13,11 +13,7 @@
 class Transform(nn.Module):
 
     # Class attribute defining transformed types. Other types are passed-through without any transformation
-    _transformed_types: Tuple[Union[Type, Callable[[Any], bool]], ...] = (
-        features.is_simple_tensor,
-        features._Feature,
-        PIL.Image.Image,
-    )
+    _transformed_types: Tuple[Union[Type, Callable[[Any], bool]], ...] = (torch.Tensor, PIL.Image.Image)
```

Comment:
As discussed offline, we can use `torch.Tensor` here directly, since `isinstance(x, features._Feature) or is_simple_tensor(x)` is the same as `isinstance(x, torch.Tensor)`. 

We should probably update the comment in the future to explain we are annotating with `Callable` now that we don't have any callable in the default definition.

Reply:
Agreed. I've added a comment.